pkg/controller/lifecycle/cnr.go

```diff
@@ -347,3 +347,9 @@ func setCNROwnerReference(cnr *apis.CustomNodeResource, node *corev1.Node) {
 		},
 	}
 }
+
+func mergeLabels(source, destination map[string]string) {
```

Comment:
there exists an function `MergeMap` in util/general.go, maybe it's better to use that instead of defining it again here

Reply:
ok, I will replace it with the general function.

Reply:
done